pdfplumber/page.py

```diff
@@ -221,6 +221,8 @@ def parse_objects(self):
         objects = {}
         for obj in self.iter_layout_objects(self.layout._objs):
             kind = obj["object_type"]
+            if kind in ["anno"]:
```

Comment:
Why not `kind == "anno"` and why `kind in ["anno"]`?

Reply:
My thinking here was that other `pdfminer.six` object types might need to be excluded in the future, and to very subtly (perhaps too subtly) indicate that through the in-array syntax. I can see the argument for making the comparison more direct, though. What do you think?

Reply:
I guess it is fine if we can expect new keys to be added here in the near future. Approving the PR. Thanks.